### ISSUE TITLE
Resourcify 1.5.1 causes Minecraft to get stuck at the loading screen when resource packs are loaded.

### ISSUE THREAD

**Issue by greyxp1** [DeDiamondPro/Resourcify#55]
Resourcify version: 1.5.1
Minecraft version: 1.20.1

The game gets stuck at the loading screen if I had any resource packs loaded before closing the game. If I disable the resource packs, the game loads fine, and it also works if I downgrade Resourcify.

**Comment by greyxp1** [comment]
This is strange. I created a new instance with the exact same mods and resource packs, and the problem disappeared. I'm not sure why downgrading Resourcify resolved the issue the first time.

**Comment by DeDiamondPro** [comment]
Yeah it’s very confusing to me to, since nothing that changed between 1.5.0 and 1.5.1 that I could think of could possible cause that.

**Comment by DeDiamondPro** [comment]
I’m going to close this issue, if you can reproduce it again and are able to tell me what is needed to reproduce the issue then feel free to reopen this issue.
